docs/sdk/sdk-by-examples/simple-governance/bridging-it-all.md

```diff
@@ -98,11 +98,6 @@ The `simplegovd` command will run the daemon server as a background process. Fir
 
 ```go
 //  cmd/simplegovd/main.go
-// SimpleGovAppInit initial parameters
```

Comment:
One small note here, I think this is getting moved in the docs update. I have a feeling that is going to merge first, so this will be a merge conflict.